Makefile

```diff
@@ -85,6 +85,18 @@ kind-init:
 	kind create cluster --config yamls/kind.yaml --name kube-ovn
 	kubectl describe no
 
+.PHONY: kind-init-cluster
+kind-init-cluster:
+	kind delete cluster --name=kube-ovn
+	kind delete cluster --name=kube-ovn1
```

Comment:
We should remove the ic-db container here and remove cluster `kube-ovn1` in `kind-clean`.

Reply:
added.